test-java-lib-lombok/src/main/java/com/uber/lombok/LombokDTO.java

```diff
@@ -32,7 +32,9 @@
   "SameNameButDifferent" /* crashes with EP 2.6.0 */,
   "InlineMeInliner" /* crashes with EP 2.7.1 */
 })
-public class LombokBuilderInit {
+public class LombokDTO {
```

Comment:
Renamed this because there is already a `LombokBuilderInit` in the testresources of the main NullAway tests.